### PR TITLE
Add ar.com.softwareperonista.Rockarrolla

### PR DESCRIPTION
**Pull request by tarmaciltur**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project.
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: ar.com.softwareperonista.Rockarrolla.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{
+    "app-id" : "ar.com.softwareperonista.Rockarrolla",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "rockarrolla",
+    "branch" : "stable",
```
Comment (hfiguiere) [inline ar.com.softwareperonista.Rockarrolla.json:7]:
```suggestion
```

File: ar.com.softwareperonista.Rockarrolla.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    "finish-args" : [
+        "--share=network",
+        "--socket=x11",
```
Comment (hfiguiere) [inline ar.com.softwareperonista.Rockarrolla.json:10]:
```suggestion
        "--socket=fallback-x11",
```

File: ar.com.softwareperonista.Rockarrolla.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--device=dri"
+    ],
+    "build-options" : {
+        "cflags" : "-O2 -g",
+        "cxxflags" : "-O2 -g",
+        "env" : {}
```
Comment (hfiguiere) [inline ar.com.softwareperonista.Rockarrolla.json:18]:
```suggestion
```

File: ar.com.softwareperonista.Rockarrolla.json [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "doctrina",
+            "buildsystem" : "meson",
+            "builddir" : true,
+            "config-opts" : [
+            ],
```
Comment (hfiguiere) [inline ar.com.softwareperonista.Rockarrolla.json:38]:
```suggestion
```

File: ar.com.softwareperonista.Rockarrolla.json [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+                {
+                    "type" : "archive",
+                    "url" : "https://gitlab.com/softwareperonista/doctrina/-/archive/0.1.13.2/doctrina-0.1.13.2.tar.gz",
+                    "sha256" : "d9863cf3a3eddbd5738509b434a949ae2a6e66de9b6687505d62cd767fe27955"
+                }
+            ]
+        },
+        {
+            "name" : "taglib",
+            "buildsystem" : "cmake",
```
Comment (hfiguiere) [inline ar.com.softwareperonista.Rockarrolla.json:49]:
```suggestion
            "buildsystem" : "cmake-ninja",
```

unless it doesn't work

File: ar.com.softwareperonista.Rockarrolla.json [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+                    "url" : "https://taglib.github.io/releases/taglib-1.12.tar.gz",
+                    "sha256" : "7fccd07669a523b07a15bd24c8da1bbb92206cb19e9366c3692af3d79253b703"
+                }
+            ]
+        },
+        {
+            "name" : "rockarrolla",
+            "buildsystem" : "meson",
+            "config-opts" : [
+            ],
```
Comment (hfiguiere) [inline ar.com.softwareperonista.Rockarrolla.json:66]:
```suggestion
```

File: ar.com.softwareperonista.Rockarrolla.json [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+    "finish-args" : [
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--device=dri"
+    ],
+    "build-options" : {
+        "cflags" : "-O2 -g",
+        "cxxflags" : "-O2 -g",
```
Comment (hfiguiere) [inline ar.com.softwareperonista.Rockarrolla.json:17]: you shouldn't need that.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build ar.com.softwareperonista.Rockarrolla

**Comment by hfiguiere** [comment]
also the PR is made against the wrong branch, it should be against `new-pr`

**Comment by tarmaciltur** [comment]
Should i close this PR and open a new one against new-pr?

File: ar.com.softwareperonista.Rockarrolla.json [hunk 8]
```diff
@@ -0,0 +1,67 @@
+{
+    "app-id" : "ar.com.softwareperonista.Rockarrolla",
```
Comment (barthalion) [inline ar.com.softwareperonista.Rockarrolla.json:2]: Just double-checking – this is an actual domain?

**Comment by barthalion** [comment]
I think it needs at least `--filesystem=xdg-music`, or I can't figure out how to use it.

File: ar.com.softwareperonista.Rockarrolla.json (hunk 8 above)
Comment (tarmaciltur) [reply ar.com.softwareperonista.Rockarrolla.json:2]: yes, it is

**Comment by tarmaciltur** [comment]
Select the music folder in configuration tab. Then go to song loader tab and click in "reload collection" button.

**Comment by barthalion** [comment]
bot, build ar.com.softwareperonista.Rockarrolla

**Pull request closed without merging**